src/main/resources/portal.properties.EXAMPLE

```diff
@@ -187,6 +187,9 @@ show.civic=false
 # Link to My Cancer Genome. Please disable (set to false) when using cBioPortal with patient identifiable data due My Cancer Genome license restrictions.
 mycancergenome.show=true
 
+# Enable transcript switch dropdown (true, false)
+show.transcript_dropdown=false
```

Comment:
```suggestion
# show.transcript_dropdown=false
```
Good to comment this one out. In case we want to make it enabled by default at some point people don't have to update their properties file

Reply:
@inodb good point, updated